app/models/reducers/summary_statistics_reducer.rb

```diff
@@ -55,6 +55,19 @@ class SummaryStatisticsReducer < Reducer
       end
     end
 
+    def summarize_field
```

Comment:
As a reviewer, it'd be really nice if for these long PRs, you minimize changes like this which just add to the lengthyness without actually changing code. Because it's batched together with actual functional changed, I have to really check if you only moved this up in the file or if there's also a subtle difference somewhere too.

Reply:
Yes, sorry about that--I honestly have no idea what caused this, perhaps a merge conflict? I don't remember even touching this file, although there were a lot of one-line changes to various model classes.

Reply:
I had to move these definitions up in the file because they were previously private, which prevented me from writing a UI to update them. It took me a while to remember because this does seem like a pointless change otherwise.